src/elements/content-uploader/UploadsManagerAction.js.flow

```diff
@@ -1,22 +1,21 @@
-/**
- * @flow
- * @file Uploads Manager action component
- */
-
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
-import messages from '../common/messages';
-import PrimaryButton from '../../components/primary-button';
-import { STATUS_ERROR } from '../../constants';
+import { injectIntl, IntlShape } from 'react-intl';
+import { Button } from '@box/blueprint-web';
+
 import { RESIN_TAG_TARGET } from '../../common/variables';
+import { STATUS_ERROR } from '../../constants';
+
+import messages from '../common/messages';
+
 import './UploadsManagerAction.scss';
 
-type Props = {
```

Comment:
in the flow file, the props should remain the same and in the tsx file should have the type -> interface, and name changes